types/v56/types.go

```diff
@@ -398,11 +398,11 @@ type AdminVdc struct {
 
 	ResourceGuaranteedMemory *float64   `xml:"ResourceGuaranteedMemory,omitempty"`
 	ResourceGuaranteedCpu    *float64   `xml:"ResourceGuaranteedCpu,omitempty"`
-	VCpuInMhz                int64      `xml:"VCpuInMhz,omitempty"`
-	IsThinProvision          bool       `xml:"IsThinProvision,omitempty"`
+	VCpuInMhz                *int64     `xml:"VCpuInMhz,omitempty"`
+	IsThinProvision          *bool      `xml:"IsThinProvision,omitempty"`
 	NetworkPoolReference     *Reference `xml:"NetworkPoolReference,omitempty"`
 	ProviderVdcReference     *Reference `xml:"ProviderVdcReference"`
-	UsesFastProvisioning     bool       `xml:"UsesFastProvisioning,omitempty"`
+	UsesFastProvisioning     *bool      `xml:"UsesFastProvisioning,omitempty"`
```

Comment:
Why do we have two pointers to Boolean and two straight Booleans in the same structure?
They are represented in the same way in XML.

Reply:
This is requeste in ref: terraform-providers/terraform-provider-vcd#380

Issue in terraform that in state file we should know if it is false or wasn't provided

Reply:
This doesn't answer the question. Why two pointers and two straight ones?

Reply:
With pointers we know that won't be returned from API when called by org admin

Reply:
I know what pointers are used for. 
My question is why we we two Booleans and two Boolean pointers in the same structure.
Why not all pointers?

Reply:
I mean. I made pointers which I know API will return or not according role, on others I don't have info

Reply:
Thanks.
FYI: the field `OverCommitAllowed` is ignored both in creation and in update.